kodilla-collections/src/main/java/com/kodilla/collections/sets/homework/StampApplication.java

```diff
@@ -16,15 +16,7 @@ public static void main(String[] args) {
 
         System.out.println(stamps.size());
         for(Stamp stamp : stamps)
-            System.out.println(stamp.getStampName() + " " + stamp.getStampSize() + " cm2");
-
-        boolean isStamped = true;
-        if (isStamped) {
-            System.out.println("Stamp is stamped.");
-
-        } else {
-            System.out.println("Stamp is not stamped.");
-        }
+            System.out.println(stamp.getStampName() + " " + stamp.getStampSize() + " cm2 " + stamp.isStamped());
```

Comment:
używaj klamerek dla pętli for zawsze.
Tutaj użyj zamiast isStamped, proponowanej wyżej metody getStamped().